validation/security-configs_test.go

```diff
@@ -81,16 +82,63 @@ func testSecurityConfigOfChain(t *testing.T, chainID uint64) {
 	)
 }
 
-func TestSecurityConfigs(t *testing.T) {
+func TestL1SecurityConfigs(t *testing.T) {
 	isExcluded := map[uint64]bool{
 		11763072: true, // Base_devnet_0 (no AnchorStateRegistryProxy specified)
 	}
 	for chainID, chain := range OPChains {
 		t.Run(perChainTestName(chain), func(t *testing.T) {
+			chain := chain
+			chainID := chainID
```

Comment:
I believe you'd want to move this above the `t.Run` if we're worried about reusing loop vars, because now you define it inside the `func` which is not guaranteed to run before the next loop iteration.
```suggestion
		chain, chainID := chain, chainID
		t.Run(perChainTestName(chain), func(t *testing.T) {
```
However, [Go 1.22. changed the behavior of loop vars](https://go.dev/blog/loopvar-preview) and they are now always a new instance and this isn't necessary any more. Are we using Go 1.22?

Reply:
Good call, fixed in [19b545c](https://github.com/ethereum-optimism/superchain-registry/pull/281/commits/19b545c3e6bcbe636d98defd6651570ccfe1f693). 

Interestingly, `go vet` did not pick up on this, I think it is a known issue that the static analysis for this problem is not perfect. Being above `t.Parallel` seemed enough to satisfy it, but I don't really understand why that should be. 

Anecdotally, I didn't actually see any problems yet with unexpected behaviour on these tests. Still, best to be safe.